packages/config/src/Config.ts

```diff
@@ -488,8 +488,28 @@ function ensureConfigHasDefaultValues({
     platforms = getSupportedPlatforms(projectRoot);
   }
 
+  const defaultIos = expWithDefaults.ios
+    ? ({
+        ...expWithDefaults.ios,
+        runtimeVersion: getRuntimeVersionNullable(expWithDefaults, 'ios') ?? undefined,
+      } as any) //TODO-JJ remove this cast in SDK 43 https://linear.app/expo/issue/ENG-1869/remove-tempruntimeversion-in-expoconfig
+    : undefined;
+  const defaultAndroid = expWithDefaults.android
+    ? ({
+        ...expWithDefaults.android,
+        runtimeVersion: getRuntimeVersionNullable(expWithDefaults, 'android') ?? undefined,
+      } as any) //TODO-JJ remove this cast in SDK 43 https://linear.app/expo/issue/ENG-1869/remove-tempruntimeversion-in-expoconfig
+    : undefined;
+
   return {
-    exp: { ...expWithDefaults, sdkVersion, platforms },
+    exp: {
+      ...expWithDefaults,
+      sdkVersion,
+      runtimeVersion: getRuntimeVersionNullable(expWithDefaults) ?? undefined,
```

Comment:
Will this throw if a policy is specified at the root level?

Even if not, I'm sort of concerned about the idea of having a field here that sometimes evaluates to the correct value and is sometimes null -- it seems like a bad idea for any other code to rely on this value. Do you have a sense of what code might use this particular field?

Reply:
Yea, good point. 

I was thinking about support for classic updates here. But thinking about it, maybe we can use the platform specific runtimes for them as well... 🤔